libraries/math/src/vector3/coreHelpers.ts

```diff
@@ -159,6 +159,66 @@ export class Vector3Utils {
         const tb = Math.sin(t * theta) / sinTheta;
         return Vector3Utils.add(Vector3Utils.scale(a, ta), Vector3Utils.scale(b, tb));
     }
+    /**
+     * multiply
+     * 
+     * Multiplies two vectors together
+     */
+    static multiply(a: Vector3, b: Vector3): Vector3 {
+        return { 
+            x: a.x * b.x, 
+            y: a.y * b.y, 
+            z: a.z * b.z 
+        };
+    }
+
+    /**
+     * rotateX
+     * 
+     * Rotates the vector around the x axis
+     * @param a Angle in radians
+     */
+    static rotateX(v: Vector3, a: number): Vector3 {
+        let cos = Math.cos(a);
+        let sin = Math.sin(a);
+        return {
+            x: v.x,
+            y: v.y * cos + v.z * sin,
+            z: v.z * cos - v.y * sin
+        }
+    }
```

Comment:
Is there a specific reason that you chose to do the rotation clockwise (i.e. left hand rule) rather than the standard right hand rule?

Preference would be to make all of these helpers adhere to the standard rotation matrix using right hand rule (counter clockwise rotation), i.e. for a 1/2 radian rotation of the unitY vector, it would become a positive unitZ vector for a X rotation rather than a negative unitZ vector

Reply:
I did it clockwise because the minecraft java function under the same name (yarn mappings) that I was familiar with also rotates clockwise. idk why they did it that way. I can make it counterclockwise if that's more standard.